strangerstudios/paid-memberships-pro
# issue 2325 Update Billing Details page shows no errors if all informa…

…tion is not added.

 * if error message container  isn't present in the DOM, create it and add to the DOM dynamically.
 * Scroll to top in order to show the message.
 
 
<img width="1376" alt="image" src="https://user-images.githubusercontent.com/1678457/234989805-b01be8e5-40c5-4cda-8eca-ae4ae9b684c6.png">
<img width="894" alt="image" src="https://user-images.githubusercontent.com/1678457/234989879-9b74e932-965b-47be-ba75-d66db71971d8.png">
<img width="725" alt="image" src="https://user-images.githubusercontent.com/1678457/234989954-45eaf125-642f-4b5b-a5e2-7e3181c557d9.png">


### All Submissions:






### Changes proposed in this Pull Request:
* if error message container  isn't present in the DOM, create it and add to the DOM dynamically.
 * Scroll to top in order to show the message.



Resolves XXX.

### How to test the changes in this Pull Request:

1. Login as admin
2. Add a stripe gateway from Settings -> Payment Gateway & SSL (don't forget to select yes to Show Billing Address Fields in PMPro Checkout Form)
3. Setup a billable level from Settings -> Levels 
4. Login as a suscriber
5. go to Membership Account
6. click view all membership options.
7.  Select a non free level
8.  Fill the form, provide poper test values and submit.
9. After success, go membership billing page/
10.  try to update any Billing Address field.
11. Check if error message is present or not.

### Other information:




### Changelog entry

> Enter a summary of all changes on this Pull Request. This will appear in the changelog if accepted.
 Restored missing error message trying to update Billing Address without providing credit card information.
